core/src/main/scala/org/apache/spark/util/collection/ExternalSorter.scala

```diff
@@ -20,6 +20,7 @@ package org.apache.spark.util.collection
 import java.io._
 import java.util.Comparator
 
+import scala.collection.BufferedIterator
```

Comment:
Change the used `scala#BufferedIterator` to `scala.collection.BufferedIterator`.